src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,34 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+from z3c.form import interfaces
+ 
+from zope import schema
+from zope.interface import Interface
+ 
+from zope.i18nmessageid import MessageFactory
+ 
+_ = MessageFactory('experimental.safe_html_transform')
+ 
+ 
+class IExperimenatalSafe_Html_TransformSettings(Interface):
+"""Global safe_html_transform settings. This describes records stored in the
+	configuration registry and obtainable via plone.registry.
+"""
+ 
+    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
```

Comment:
Yeah I know what is going on there just want to conform that either we go
with current structure or we should change according to the commit one. But
yeah got my answer. Will correct that :)

Prakhar Joshi
DA-IICT,Gandhinagar

On Mon, Feb 9, 2015 at 10:41 PM, Timo Stollenwerk <notifications@github.com>
wrote:

> In src/experimental/safe_html_transform/interfaces.py
> <https://github.com/collective/experimental.safe_html_transform/pull/1#discussion_r24345111>
> :
>
> > +from z3c.form import interfaces
> > +
> > +from zope import schema
> > +from zope.interface import Interface
> > +
> > +from zope.i18nmessageid import MessageFactory
> > +
> > +_ = MessageFactory('experimental.safe_html_transform')
> > +
> > +
> > +class IExperimenatalSafe_Html_TransformSettings(Interface):
> > +"""Global safe_html_transform settings. This describes records stored in the
> > +	configuration registry and obtainable via plone.registry.
> > +"""
> > +
> > +    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
>
> It doesn't matter in Plone if those files are in a different folder. You
> can keep your current structure. You just have to replace the Class.
> Though, you need to understand what you are doing. If you are just
> copy/pasting code it won't work. See
> https://pypi.python.org/pypi/plone.app.registry for details about the
> registry control panels.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/collective/experimental.safe_html_transform/pull/1/files#r24345111>
> .
>